File: src/main/java/no/nav/veilarboppfolging/controller/v2/request/AvsluttOppfolgingV2Request.java
package no.nav.veilarboppfolging.controller.v2.request;


import no.nav.common.types.identer.Fnr;
import no.nav.common.types.identer.NavIdent;

public class AvsluttOppfolgingV2Request {
    NavIdent veilederId;
    String begrunnelse;
    Fnr fnr;
}

File: src/main/kotlin/no/nav/veilarboppfolging/controller/v2/request/AvsluttOppfolgingV2Request.kt
package no.nav.veilarboppfolging.controller.v2.request

import no.nav.common.types.identer.Fnr
import no.nav.common.types.identer.NavIdent


data class AvsluttOppfolgingV2Request(
    val veilederId: NavIdent,
    val begrunnelse: String,
    val fnr: Fnr,
)
